frame/example/src/lib.rs

```diff
@@ -292,7 +310,9 @@ impl<T: pallet_balances::Config> WeighData<(&BalanceOf<T>,)> for WeightForSetDum
 {
```

Comment:
re: https://github.com/paritytech/substrate/pull/8301#discussion_r600382729

In here, we start implementing the WeightData, ClassifyDispatch, and PaysFee. In each implementations, we do use the paramter `(&BalanceOf<T>)`. Is that what you mean to "use it" in your comment?